selfdrive/car/subaru/values.py

```diff
@@ -41,7 +41,7 @@ class SubaruCarInfo(CarInfo):
 
 
 CAR_INFO: Dict[str, Union[SubaruCarInfo, List[SubaruCarInfo]]] = {
-  CAR.ASCENT: SubaruCarInfo("Subaru Ascent 2019-20", "All"),
+  CAR.ASCENT: SubaruCarInfo("Subaru Ascent 2019-20", "All", good_torque=True),
   CAR.IMPREZA: [
```

Comment:
no half star for the impreza?

Reply:
Impreza has `0.8234374840709592` of measured accel in the old and new data

Reply:
Looking at some segments, it's hitting around 1.5 with no steering pressed